model/kubernetes/kubernetes_cluster.rb

```diff
@@ -0,0 +1,65 @@
+#  frozen_string_literal: true
+
+require_relative "../../model"
+
+class KubernetesCluster < Sequel::Model
+  one_to_one :strand, key: :id
+  one_to_one :api_server_lb, class: :LoadBalancer, key: :id, primary_key: :api_server_lb_id
+  many_to_one :private_subnet
+  many_to_one :project
+  many_to_many :cp_vms, join_table: :kubernetes_clusters_cp_vms, class: :Vm, order: :created_at
+
+  dataset_module Pagination
+
+  include ResourceMethods
+  include SemaphoreMethods
+  include Authorization::HyperTagMethods
+
+  semaphore :destroy
+
+  def display_location
+    LocationNameConverter.to_display_name(location)
+  end
+
+  def hyper_tag_name(project)
+    "project/#{project.ubid}/location/#{display_location}/kubernetes-cluster/#{name}"
+  end
+
+  def path
+    "/location/#{display_location}/kubernetes-cluster/#{name}"
+  end
+
+  def endpoint
+    api_server_lb.hostname
+  end
+
+  def disassociate_cp_vm(vm)
```

Comment:
When you define a `many_to_many` association, `remove_*` methods are added automatically. `remove_cp_vm` might perform the same function. cc @jeremyevans as an expert

https://github.com/jeremyevans/sequel/blob/master/doc/association_basics.rdoc#label-Methods+Added

Reply:
thanks!